nservicebus/azure-servicebus/publisher-names-configuration.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Publishers name configuration
+summary: Configuration mapping between publisher names and message types for Endpoint Oriented Topology
+tags:
+- Cloud
+- Azure
+- Transports
+---
+
+`ITopology` interface exposes a property (`NeedsMappingConfigurationBetweenPublishersAndEventTypes`) to instruct transport environment if mapping must be configured. Custom topologies have to properly configure this property.   
```

Comment:
@mikeminutillo  I did it, but @weralabaj suggested to me to move this paragraph at the beginning...I'll move it back